### DISCUSSION TITLE
Using Munify for a local conference

### DISCUSSION THREAD

**Discussion by Scrqm** [DeutscheModelUnitedNations/munify#205] [Support]
Hello!
I got Munify running on my local machine, and it seems to work quite well (I can create conferences, manipulate committee info, etc). I had a few questions, however:

1. How should I add new organizers to my conferences via the web portal? I can probably just edit the database schema directly, but not everyone on my conference team is comfortable with database stuff/tech and they would like to add/remove chairs, participants, etc
2.  There are some issues regarding the Speakers list (the app crashes whenever there is no speaker for instance) that could pose an issue

The app seems very great and polished right now, far better than other free MUN app alternatives. Thank you so much for your work!

**Comment by Strehk** [comment]
Hey @Scrqm,
thanks so much for getting in touch and we are very happy to see that our project is attracting attention outside of our MUN-Realm.

May I ask what MUN you are planning to use it for and what the timeframe is? We are happy to help you getting set up or even host the software for you if that is sensible.

Since this is the first request of someone not from our organization wanting to use the software, there are still some hard-coded parts directly related to our organization and our MUNs, so be aware of that – but we will certainly try to remove as many of those as possible for you if you actually plan to use CHASE in production. Also, interaction with delegates have not yet been tested, we only used it in production for the chairs until now.

**Comment by Strehk** [comment]
To answer your questions:

1. To add staff, you – as of now – actually need to go into the database and connect users to conferences as teamMembers manually. Or we also have an Endpoint in the API that lets you bulk upcert users by there email and connects them with a given conference. Your new users just need to reset its passwords and and set a new one and they are good to go.

2. Oh, that obviously should not happen. Please make sure that you have an agenda item set – the speakers lists hinge on an agenda item, so if there is no item set, the speakers list may crash (the error handling on that part could arguably be better, but since we always prefilled an agenda item for each committee, we didn't have that issue). Let me know if that solves the issue – if not, please open an issue.

**Comment by Strehk** [comment]
And lastly just a quick disclaimer:
MUNify CHASE is part of the greater Project MUNify, which aims to cover all aspects of MUN Organization. Currently, we are working hard on [MUNify Delegator](https://github.com/DeutscheModelUnitedNations/munify-delegator), which is a registration Platform for (our) MUNs. Maybe that is of interest to you as well – but I am telling you this because in order to streamline the puzzle peaces of software, we will remove the login and registration flow from CHASE until March 2025, since we are using an OIDC Issuer ([Zitadel](https://zitadel.com/) to be specific) as an centralized account hub. Please bear that in mind – if you plan on using CHASE in Production, depending on the timeframe I would suggest to also plan with the OIDC approach for logging in users.
